dashboard/src/components/Header/Menu.tsx

```diff
@@ -86,6 +86,10 @@ function Menu({ clusters, appVersion, logout }: IContextSelectorProps) {
                 .
                 <br />
                 {appVersion}
+                <br />
+                <Link to={"/docs"} key="deploy-button">
```

Comment:
why the `key`?

Reply:
My bad!  unnoticed during copypasting :S